plugins/main/public/components/common/welcome/components/sca_scan/sca_scan.tsx

```diff
@@ -277,7 +277,7 @@ export const ScaScan = compose(
                         href={getCore().application.getUrlForApp(
                           configurationAssessment.id,
                           {
-                            path: `#/overview?tab=sca&redirectPolicy=${lastScan.policy_id}`,
+                            path: `#/overview?tab=sca&redirectPolicy=${lastScan?.policy_id}`,
```

Comment:
**thought**: This solves the problem if the `lastScan` variable is undefined or something like that, but for this cause, the button will redirect to `#/overview?tab=sca&redirectPolicy=undefined` that doesn't make sense and could cause problems because the `redirectPolicy` query parameter causes a specific policy is selected.